<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GSRItem</class>
 <widget class="QWidget" name="GSRItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>150</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GSRItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: #ffffff;</string>
  </property>
  <widget class="QPushButton" name="m_avatarPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_nameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>7</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>博客交流群</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_memberCountLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2000人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_addPushButton">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>30</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8fc0e7;</string>
   </property>
   <property name="text">
    <string>+添加</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
